cmd/syncthing/gui.go

```diff
@@ -233,8 +233,8 @@ func (s *apiService) Serve() {
 	getRestMux.HandleFunc("/rest/db/need", s.getDBNeed)                          // folder [perpage] [page]
 	getRestMux.HandleFunc("/rest/db/status", s.getDBStatus)                      // folder
 	getRestMux.HandleFunc("/rest/db/browse", s.getDBBrowse)                      // folder [prefix] [dirsonly] [levels]
-	getRestMux.HandleFunc("/rest/events", s.getIndexEvents)                      // since [limit]
-	getRestMux.HandleFunc("/rest/events/disk", s.getDiskEvents)                  // since [limit]
+	getRestMux.HandleFunc("/rest/events", s.getIndexEvents)                      // since [limit] [timeout]
+	getRestMux.HandleFunc("/rest/events/disk", s.getDiskEvents)                  // since [limit] [timeout]
```

Comment:
> While I'm here, is it worth reducing the size of the disk events buffered subscription to 25, as we only ever fetch 25 events? Or not worth the effort?

When I wrote that code I made the API buffer 1000 events and just use the first 25.  My thoughts were that maybe someone else could use more info if they wanted to include a feature that used more.  Or something else I just hadn't thought of.  I was future proofing, but I have no problem with you limiting it to just 25 if you feel that's better.